guides/hack/11-built-in-types/35-enum-class.md

```diff
@@ -136,6 +136,35 @@ enum class E1 : IBox extends E {
 // only he name is use for ambiguity
 ```
 
+### Diamond shape scenarios
+Enum classes support diamond shaped inheritance as long as there is no ambiguity, like int:
```

Comment:
just wondering if you're saying that diamond inheritance (like in this example) only work for built-in types? Everything else is clear!